pootle/static/js/shared/utils/fetch.js

```diff
@@ -29,7 +29,7 @@ function fetch({ url, body, method = 'GET', dataType = 'json', queue = null,
       data: body,
     })
   );
-
+  requests[queueName].done(() => delete requests[queueName]);
```

Comment:
Actually, [after a read on `delete`](http://bertanguven.com/preventing-memory-leaks-in-javascript-null-vs-delete), it seems like we'd prefer to nullify the value `requests[queueName] = null`.